magpie/adapter/magpieowssecurity.py

```diff
@@ -87,8 +87,7 @@ def update_request_cookies(self, request):
                                          .format(session_resp.reason))
 
             # use specific domain to differentiate between `.{hostname}` and `{hostname}` variations if applicable
-            # noinspection PyProtectedMember
-            request_cookies = session_resp.request._cookies
+            request_cookies = session_resp.request._cookies  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _cookies of a client class](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)